addon/utils/sortable-by-position.js

```diff
@@ -0,0 +1,25 @@
+/**
```

Comment:
i :heart: this refactor.

Reply:
Me too, for realzieees. I was playing around with decorators when I realized it didn't need to be that hard!